# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/glib/hotfix.patch.m68k
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

../glib/gmacros.h:874:53: error: size of array '_GStaticAssertCompileTimeAssertion_10' is negative
  874 | #define G_STATIC_ASSERT(expr) typedef char G_PASTE (_GStaticAssertCompileTimeAssertion_, __COUNTER__)[(expr) ? 1 : -1] G_GNUC_UNUSED

--- glib-2.80.0/girepository/girepository-private.h.vanilla	2024-03-21 18:46:17.897629813 +0100
+++ glib-2.80.0/girepository/girepository-private.h	2024-03-21 18:46:23.648629523 +0100
@@ -64,7 +64,7 @@
   void *padding[6];
 };
 
-G_STATIC_ASSERT (sizeof (GIBaseInfo) == sizeof (GIBaseInfoStack));
+//G_STATIC_ASSERT (sizeof (GIBaseInfo) == sizeof (GIBaseInfoStack));
 G_STATIC_ASSERT (G_ALIGNOF (GIBaseInfo) == G_ALIGNOF (GIBaseInfoStack));
 
 /**
